src/internal/operations/restore.go

```diff
@@ -68,7 +68,7 @@ func NewRestoreOperation(
 		operation:  newOperation(opts, bus, kw, sw),
 		acct:       acct,
 		BackupID:   backupID,
-		RestoreCfg: restoreCfg,
+		RestoreCfg: control.EnsureRestoreConfigDefaults(ctx, restoreCfg),
```

Comment:
Isin't this something that we specify in code? What is the possibility that we get the incorrect value? Also, if we do get incorrect value, wouldn't it be better to error out and force us to fix it instead of silently doing potentially the incorrect thing.

Reply:
I think the larger likelihood is that we get an unpopulated value, and want to ensure it gets set to a default instead of remaining unpopulated.  Yes, this pattern does silently fix bad values without much alert. But since we 1/ export consts for enums and 2/ don't consider other values "improper", I think the potential ramifications here are at a minimum.  A customer would have to deliberately go out of their way and feed in obviously unsupported data for that to happen, which is not likely.

In other situations, where it could be easy for a customer to accidentally feed unsupported values into the configuration, I agree that returning an error is an appropriate pattern.